src/main/java/frc/robot/subsystems/drive/SwerveModules.java

```diff
@@ -87,7 +88,7 @@ public double getDriveVelocityMPerS() {
     }       
```

Comment:
The getDriveVelocityMPerS isn't used as far as I know, also the comment above the function is wrong.

In any case, this is converting RPM to m/s so we need to divide by 60 to get this into seconds.

Reply:
Done